patches/components-permissions-BUILD.gn.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
+index 799755985b575a7464897d0969bf383c386d267c..80a7190f344b27a58c8c05b710a1b19b292031eb 100644
+--- a/components/permissions/BUILD.gn
++++ b/components/permissions/BUILD.gn
+@@ -2,6 +2,8 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/components/permissions/sources.gni")
++
+ source_set("permissions_common") {
+   sources = [
+     "features.cc",
+@@ -89,6 +91,8 @@ source_set("permissions") {
+     "//ui/base",
+     "//url",
+   ]
++  sources += brave_components_permissions_sources
```

Comment:
there is a policy to make one line patches, see for example components-omnibox-browser-BUILD.gn.patch

Reply:
yeah... I saw these one-liners in ```*.gn``` patches, it greatly reduces readability imo. but a policy is a policy. will fix that.

Reply:
You can reduce it even more by adding import to the same line. For example: https://github.com/brave/brave-core/blob/master/patches/components-sync-protocol-protocol_sources.gni.patch

Reply:
@goodov the readability of the gn patches is really not a big concern and most of them are pretty standardized anyway. The maintenance of these patches on chromium updates is a much bigger concern and fewer lines changed leads to fewer conflicts